data.sql

```diff
@@ -5,3 +5,13 @@ VALUES (2, 'Gabumon', '2018-11-15', 2, TRUE, 8.0),
 (3, 'Pikachu', '2021-07-01', 1, FALSE, 15.04),
 (4, 'Devimon', '2017-05-17', 5, TRUE, 11);
 
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES(5, 'Charmander', '2020-02-08', 0, FALSE, -11),
+(6, 'Plantmon', '2021-11-15', 2, TRUE, -5.7 ),
+(7, 'Squirtle', '1993-04-02', 3, FALSE, -12.13),
+(8, 'Angemon', '2005-06-12', 1, TRUE, -45 ),
```

Comment:
- Please, kindly add a  `dot` and zero to the numbers as they're floating numbers to meet the project requirement. `-45.0`